packages/hash/frontend/src/components/hooks/useGetAccountIdForShortname.ts

```diff
@@ -12,18 +10,6 @@ export const useGetAccountIdForShortname = (
   const { loading: orgsLoading, orgs } = useOrgs(true);
 
   const accountId = useMemo(() => {
-    /**
-     * @todo - This is incredibly flakey right now.
-     *    We should be creating system types and entities under the actual org, _not_ the nilUuid
-     *    https://app.asana.com/0/1202805690238892/1203427737626951/f
-     */
-    if (
-      shortname === SYSTEM_ACCOUNT_SHORTNAME ||
-      shortname === "example" // This is the root account
```

Comment:
Quick question, how do we refer to the built-in link-type that is created in the graph layer? I assume that since it's not a system type but a builtin that it would have the nil-uuid as the owner, which we don't have a shortname for. Do we not make it browsable/addressable on purpose?

Reply:
> Do we not make it browsable/addressable on purpose?

Correct, somewhat in the same way that there's no way to discover the primitive data types in the frontend (outside of selecting them for values, I mean you can't actually _view_ them). This will all have to change when we implement a data type editor, and inheritance but for now it holds true.

Also the primitive types will need special casing anyway as their URLs will be blockprotocol ones, which means we won't have normal shortname-based routing anyway I think (so the nil-uuid should end up irrelevant and not exposed)

Reply:
This is a good point which I hadn't considered. I don't think we are currently exposing this in the browser, but it's a good point that there is still technically an entity type which has the `nilUuid`. Not sure whether we care about this.

cc @Alfred-Mountfield